### ISSUE TITLE
Feature Request : Quick HTML palette export

### ISSUE THREAD

**Issue by didli** [thezbyg/gpick#126]
Hi ! As I was trying to show a new color palette to a friend of mine, I realized it could be useful to be able to share/export a html page with the selected palette (based on the grid view, for example, since we already have CSS export).
What do you think ^^ ? Is it possible ?

**Comment by thezbyg** [comment]
Hi,
Yes it is possible.
Do you need any additional features (grid item size control, changeable page background color,...) in the exported HTML file?

Preview screenshot:
![html_example](https://cloud.githubusercontent.com/assets/129256/14031479/acb57ff2-f215-11e5-80da-039942dc5ec3.png)

Preview HTML page:
[http://static.gpick.org/examples/html_example.html](url)

**Comment by didli** [comment]
Wow ... oO
Amazing ! Your preview rocks. That's exactly what I was thinking about. 
Now I didn't thought that far in this feature, but yes I think grid size control could be quite useful ^^. Hmm... then maybe an easy way to copy these colors (web hex code ?) from a click on each color in the html ? Anyway, thanks for the update. I can't wait to see your ideas about this feature.

**Comment by thezbyg** [comment]
Added grid size control, background color selection and copy-on-click on web hex code elements. At export time the following options are available:
 * Grid item size (small, medium, big or dynamically controllable).
 * Background color (white, gray, black, first color, last color or dynamically selected).
 * Include/exclude color names.

New preview screenshot:
![html_example2](https://cloud.githubusercontent.com/assets/129256/14062648/0cbea0aa-f3af-11e5-9703-b796b7a2d11d.png)

New preview HTML page:
[http://static.gpick.org/examples/html_example2.html](http://static.gpick.org/examples/html_example2.html)

**Comment by didli** [comment]
This is good. This is damned good. And that was fast. 
Now I could even use this feature as a quick way to recall colors for other opened apps. The html export is in a single file ?

**Comment by thezbyg** [comment]
Yes, a single file is exported without any dependencies or additional files.
